From: Etienne Millon <etienne.millon@gmail.com>
Date: Sun, 18 Dec 2011 22:56:36 +0100
Subject: Fix typo separated

Glyr source code contains several occurrences of a typo on "separated".
---
 lib/glyr.h                  |    2 +-
 lib/intern/lyrics/elyrics.c |    2 +-
 src/glyrc/glyrc.c           |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/glyr.h b/lib/glyr.h
index 48a3ac7..1627c74 100644
--- a/lib/glyr.h
+++ b/lib/glyr.h
@@ -558,7 +558,7 @@ GLYR_ERROR glyr_opt_plugmax(GlyrQuery * s, int plugmax);
 /**
 * glyr_opt_allowed_formats:
 * @s: The GlyrQuery settings struct to store this option in.
-* @formats: A commaseperated list of allowed formats.
+* @formats: A comma-separated list of allowed formats.
 *
 * Restricts providers to retrieve at max. @plugmax items, you might use this to get results
 * over several providers when glyr_opt_number() is set to something higher than 1.
diff --git a/lib/intern/lyrics/elyrics.c b/lib/intern/lyrics/elyrics.c
index c4cec17..ab59852 100644
--- a/lib/intern/lyrics/elyrics.c
+++ b/lib/intern/lyrics/elyrics.c
@@ -54,7 +54,7 @@ static const gchar * lyrics_elyrics_url(GlyrQuery * settings)
 #define FROM_END   "<br><br>"
 #define BAD_STRING "Lyrics removed for copyright protection!"
 
-/* This data is seperated from the actual lyrics => remove it from here */
+/* This data is separated from the actual lyrics => remove it from here */
 static void remove_from_from_string(gchar * string)
 {
     gchar * from_middle = strstr(string,FROM_MIDDLE);
diff --git a/src/glyrc/glyrc.c b/src/glyrc/glyrc.c
index 0053d1e..10e844c 100644
--- a/src/glyrc/glyrc.c
+++ b/src/glyrc/glyrc.c
@@ -183,7 +183,7 @@ void help_short(GlyrQuery * s)
             IN"-t --title            String: Songname to search for\n"
             IN"-e --maxsize          Integer: (images only) The maximum size a cover may have.\n"
             IN"-i --minsize          Integer: (images only) The minimum size a cover may have.\n"
-            IN"-F --formats          String: A semicolon seperated list of imageformats that are allowed. e.g.: \"png;jpeg\"\n"
+            IN"-F --formats          String: A semicolon separated list of imageformats that are allowed. e.g.: \"png;jpeg\"\n"
             IN"-8 --force-utf8       Forces utf8 encoding for text items, invalid encodings get sorted out\n"
             "\nDATABASE OPTIONS\n"
             IN"-c --cache <folder>   Creates or opens an existing cache at <folder> and lookups data from there.\n"
-- 
